.zuul.yaml

```diff
@@ -0,0 +1,9 @@
+---
+- project:
+    name: packit-service/packit
```

Comment:
Also note that this line is not mandatory. As this is a zuul configuration file in a Zuul untrusted repository then by default the "project" statement is bound this project. But having that line is fine too :) it just have no effect :)